api/.env.development

```
@@ -0,0 +1 @@
+engine_connect_args = {"check_same_thread": false}
```

```suggestion
engine_connect_args = {"check_same_thread": False}
```
Dont know if we want python-style here, but if no just ignore this comment and merge if you want

---

With the big F, it's no longer a valid json and the whole app will just crash on startup. So let's leave it^^